### DISCUSSION TITLE
Uso de gitignore

### DISCUSSION THREAD

**Discussion by notyourlasagna** [IIC2133-PUC/2024-1#37] [Tarea 0] [answered]
Hola! Debemos utilizar gitignore? en este caso, qué debemos omitir?

**Comment by PabloNavarroY** [comment, marked as answer]
Puedes modificar el .gitignore de cualquier manera que gustes y que no afecte a tu evaluación. Con esto último me refiero a que no ignores cosas importantes como el `Makefile` ni tu código fuente ubicado en `src`.

El .gitignore es una utilidad. No "tienes" que ocuparlo, pero sí puedes ocuparlo si es que quieres.
